### PR TITLE
adding hero section

### PR DESCRIPTION
**Pull request by sergey221997**

### CODE REVIEW

File: sections/hero.liquid
Comment (MKhakimov) [inline sections/hero.liquid:1]: image

**Review by MKhakimov — CHANGES_REQUESTED** [review]
Should be blocks

File: sections/hero.liquid [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% for block in section.blocks %}
+  <h1>{{ block.settings.heading }}</h1>
+  <h2>{{ block.settings.subheading }}</h2>
+  <h3>{{ block.settings.text }}</h3>
```
Comment (AlexBeskrovniy) [inline sections/hero.liquid]: Use 'case' operator to resolve which content you need to show according to block type. For example, only one block has setting with 'heading' ID, but you are trying to render it for every block. I can't see the demo but I'm sure there are a lot of empty "h1" tags

**Review by AlexBeskrovniy — CHANGES_REQUESTED** [review]
Use 'case' operator to resolve which content you need to show according to block type. For example, only one block has setting with 'heading' ID, but you are trying to render it for every block. I can't see the demo but I'm sure there are a lot of empty "h1" tags

File: sections/hero.liquid [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+    {% when 'subheading' %}
+      <div>
+        {{ block.settings.subheading }}
+      </div>
+    {% when 'text' %}
+      <div>
+        {{ block.settings.text }}
+      </div>
+  {% endcase %}
 {% endfor %}
```
Comment (AlexBeskrovniy) [inline sections/hero.liquid]: Cool, but I think we need to have our own wrapper for section content. And use semantic tags for blocks content.

**Review by AlexBeskrovniy — APPROVED** [review]

**Review by MKhakimov — APPROVED** [review]

**Pull request merged**